packages/manager/apps/pci-rancher/src/components/layout-helpers/Dashboard/Dashboard.component.tsx

```diff
@@ -51,7 +50,8 @@ const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs, rancher }) => {
   const { projectId, rancherId } = useParams();
   const { data: versions } = useVersions();
   const { t } = useTranslation('dashboard');
-  useTrackingPage(TrackingPageView.DetailRancher);
+  const { trackCurrentPage } = useOvhTracking();
+  trackCurrentPage();
```

Comment:
Shouldn't this be in a useEffect ? it might be call within each render

Reply:
you can delete the trackCurrent page here and only use it on Layout.component.tsx

Reply:
Yes and you can remove it since it is already done once in the Layout.component.tsx and trigger on location change